src/resources/js/views/manager.js

```diff
@@ -646,9 +647,22 @@ tribe.events.views.manager = {};
 		// Update the global set of containers with all of the manager object.
 		obj.selectContainers();
 
-		// Trigger the browser pushState
+		// Trigger the browser pushState.
 		obj.updateUrl( $container );
 
+		// Focus on either the events list or the no results message for improved accessibility.
+		var $eventsList = $container.find('.tribe-events-calendar-list');
+		var $noResults = $container.find('.tribe-events-c-messages__message--notice');
+				
+		if ($eventsList.length && $eventsList.find('.tribe-events-calendar-list__event-row').length) {
+			$eventsList
+				.attr('tabindex', '-1')
+				.attr('aria-label', tribe_events_views_strings.listOfEvents || 'List of Events')
+				.trigger('focus');
+		} else if ($noResults.length) {
+			$noResults.attr('tabindex', '-1').trigger('focus');
+		}
+
```

Comment:
Move this out of the Core Manager, into hooking into `afterAjaxSuccess`.

Use ES6, and avoid using global variables like `tribe_events_views_strings`, pull strings from the `data-` params. And avoid a default value in this case.

Spacing doesn't match our standards.